1_space_odyssey/js/mannuel-9000/app.js

```diff
@@ -85,25 +85,25 @@ _______/\\\\\________/\\\\\\\\\\\\______/\\\________/\\\______/\\\\\\\\\\\______
     b. ensure to choose the correct values and datatypes
     c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
 */
-var isAlive;
```

Comment:
please make folder of your name and create an app.js and put all this code in there.
`js/erik-9000/app.js`